### PR TITLE
Update Pod Design section

### PR DESCRIPTION
**Pull request by akalex**
Add one more exercise for CronJob.

### CODE REVIEW

File: c.pod_design.md [hunk 1]
```diff
@@ -728,3 +728,47 @@ kubectl delete cj busybox
...
+metadata:
+  creationTimestamp: null
+  name: time-limited-job
+spec:
+  jobTemplate:
+    metadata:
+      creationTimestamp: null
+      name: time-limited-job
+    spec:
+      activeDeadlineSeconds: 17 # add this line
```
Comment (dgkanatsios) [inline c.pod_design.md:755]: This should be 30,no?

Comment (akalex) [reply c.pod_design.md:755]: Should be 17. I will update description

Comment (akalex) [reply c.pod_design.md:755]: Fixed

**Comment by dgkanatsios** [comment]
thank you!

**Pull request merged**